src/main/resources/css/common.css

```diff
@@ -147,7 +151,6 @@ span {
 }
 .unavailable {
     background: white !important;
-    border: none !important;
```

Comment:
we removed this border style from here and they ended up adding the same border: none and two different places with, def-card.unavailable and chaptercard.unavailable. I think it would be better if we just had it here common no? because we aren't doing anything else with chaptercard.unavailable and def-card.unavailable aside from setting border:none